dist/index .html

```diff
@@ -0,0 +1,359 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="Css/style.css" />
+    <link rel="stylesheet" href="Css/queries.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&family=Playfair+Display&display=swap"
+      rel="stylesheet"
+    />
+    <title>Document</title>
+  </head>
+  <body>
+    <section class="home-page">
+      <header>
+        <ul id="headerList">
+          <li class="mobile menu_">
+            <a
+              ><img
+                src="img/mobile-img/menu-outline.svg"
+                id="menuButton"
+                alt="menu-button"
+            /></a>
+          </li>
+          <li class="Desktop facebook">
+            <a><p>f</p></a>
+          </li>
+          <li class="Desktop">
+            <a
+              ><img
+                class="twitter"
+                src="img/Desktop-img/twitter.png"
+                alt="twiiter logo"
+            /></a>
+          </li>
+          <li class="Desktop">
+            <a><p class="languages">English</p></a>
+          </li>
+          <li class="Desktop">
+            <a><p class="languages">My page</p></a>
+          </li>
+          <li class="Desktop">
+            <a><p class="languages">Logout</p></a>
+          </li>
+        </ul>
+        <div class="menu hidden mobile">
+          <ul>
+            <li class="home"><a>Home</a></li>
+            <li class="about"><a>About</a></li>
+            <li><a class="buyTicket">Join us</a></li>
+          </ul>
+        </div>
+        <div class="menu Desktop">
+          <ul class="menuDesk">
+            <li>
+              <a
+                ><img
+                  src="img/poth-version-img/large.png"
+                  class="LogoDesktop"
+                  alt="Logo for desktop"
+              /></a>
+            </li>
+            <li class="home"><a>Home</a></li>
+            <li class="about"><a>About</a></li>
+            <li><a class="buyTicket">Join us</a></li>
+          </ul>
+        </div>
+      </header>
+
+      <div class="container">
+        <p class="display-1">"Hello! Sharing world"</p>
+        <h1 class="display-1 fw-bold">
+          Global Designers <br />summit<span class="display-1 fw-bold"
+            >2021</span
+          >
```

Comment:
### original design
![original-hero](https://user-images.githubusercontent.com/80149314/142506542-914eb32c-0f50-47c4-b2c7-e7c90e7c64db.jpg)

### your design
![review-hero](https://user-images.githubusercontent.com/80149314/142506855-3ad5bbc6-4017-43d3-95f7-210e4fb13b49.jpg)

Please make sure your deign looks the same as the given design. Your navbar,the dates and the text need some work. The font size should be big enough like in the given mockup.